ArchiSteamFarm/Localization/Strings.resx

```diff
@@ -661,4 +661,11 @@ StackTrace:
   <data name="TargetBotNotConnected" xml:space="preserve">
     <value>Target bot instance is not connected!</value>
   </data>
-</root>
+  <data name="BotWalletBalance" xml:space="preserve">
+    <value>Wallet balance: {0} {1}</value>
+    <comment>{0} will be replaced by wallet balance value, {1} will be replaced by currency name</comment>
+  </data>
+  <data name="BotHasNoWallet" xml:space="preserve">
+    <value>Bot has no wallet.</value>
+  </data>
+</root>
```

Comment:
No newline at end of file - old version had one and .editorconfig says `insert_final_newline = true`

Reply:
This file auto generated by VS.

> .editorconfig says insert_final_newline = true
It's VS settings?

Reply:
yes and no. [.editorconfig](https://github.com/JustArchiNET/ArchiSteamFarm/blob/master/.editorconfig) is a file in ASFs main directory. normally VS should respect the code-formatting settings in there. maybe your VS isn't configured that way. IIRC following setting should be the right one if you want to check 😉 


![image](https://user-images.githubusercontent.com/6608231/47955346-0fa00080-df97-11e8-82d6-d3445061fff2.png)

Reply:
@Abrynos VS doesn't always respect this, but it's fine, I can fix it myself.

Reply:
![image](https://user-images.githubusercontent.com/31675199/47955363-da4be080-df9f-11e8-8ea7-d5783f571ae7.png)
It does not work for me.

Reply:
Well... you know who coded VS 😉 what do you expect xD that's why i proposed using `error` instead of `warning` with .editorconfig, because that gets respected EVERYTIME (since it's checked before compiling). Logically this is wrong because it's not a compiler-error and it shouldn't prevent ocmpiling but it's the only way to force it

Reply:
I do not get a `warning`.
It looks like `insert_final_newline = true` does not work in `* .resx` files (I checked in `* .cs` works well)